src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -34,7 +35,19 @@
 
 
                 <controls:SettingsGroup x:Uid="Shortcut" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}">
-                    <controls:SettingExpander IsExpanded="True">
+
+
+                    <controls:Setting x:Uid="PowerLauncher_IsWindowsSearchReplacementMode">
+                        <controls:Setting.ActionContent>
+                            <ComboBox SelectedIndex="{x:Bind Mode=TwoWay, Path=ViewModel.IsWindowsSearchReplacementMode, Converter={StaticResource BoolToComboBoxIndexConverter}}" MinWidth="{StaticResource SettingActionControlMinWidth}">
+                                <ComboBoxItem x:Uid="Run_Radio_WindowsSearchAddOnMode"/>
+                                <ComboBoxItem x:Uid="Run_Radio_WindowsSearchReplacementMode"/>
+                            </ComboBox>
+                        </controls:Setting.ActionContent>
+                    </controls:Setting>
+
+
+                    <controls:SettingExpander IsExpanded="True" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsWindowsSearchReplacementMode, Converter={StaticResource BoolNegationConverter}}">
```

Comment:
Use hide property instead of enabled property to be in line with shortcut guide and others where we hide the controls.

Reply:
Update. including above screenshots.